django-backend/fecfiler/committee_accounts/utils.py

```diff
@@ -105,14 +105,23 @@ def get_committee_account_data(committee_id):
 """
 
 
-def query_fec_api(endpoint, params):
+def query_fec_api_single(endpoint, params):
```

Comment:
EXCELLENT job refactoring this!  You improved the code using it and the readability.  I especially like renaming `json_results` to `fec_api_candidates`.  It captures what the variable is.

Since this is now used in both the committee account module and the contact module, let's move it to the shared module.